contractTest/gateways/MsSQL/deleteVisitByCallId.contractTest.js

```diff
@@ -0,0 +1,65 @@
+import deleteVisitByCallIdGateway from "../../../src/gateways/MsSQL/deleteVisitByCallId";
+import {
+  setupVisit,
+  setupOrganisationFacilityDepartmentAndManager,
+} from "../../../test/testUtils/factories";
+import AppContainer from "../../../src/containers/AppContainer";
+
+describe("deleteVisitByCallIdGateway", () => {
+  const container = AppContainer.getInstance();
+
+  it("cancels a visit", async () => {
+    // Arranges
+    const {
+      departmentId,
+    } = await setupOrganisationFacilityDepartmentAndManager();
+    const { uuid: callId } = await setupVisit({ wardId: departmentId });
```

Comment:
You can now use setUpScheduledCall() in factories
const { uuid } = await setUpScheduledCall({ departmentId });